kimchi_grpc_server/kimchi_grpc_server/kimchi_pb2_grpc.py

```diff
@@ -138,6 +158,30 @@ def SendSelectedPose(self, request, context):
         context.set_details('Method not implemented!')
         raise NotImplementedError('Method not implemented!')
 
+    def ContinuePath(self, request, context):
+        """Missing associated documentation comment in .proto file."""
```

Comment:
this comment is intended as a reminder to add the documentation comment later? if so please add a TODO.